cyhomework1.cy.js

```diff
@@ -0,0 +1,101 @@
+describe('Register User Test', () => {
+    it('should register a new user successfully', () => {
+        // 1. Launch browser and navigate to the URL
+        cy.visit('http://automationexercise.com');
+
+        // 2. Verify that home page is visible successfully
+        cy.get('body').should('be.visible');
+
+        // 3. Click on 'Signup / Login' button
+        cy.get('header > div > div > div > div.col-sm-8 > div > ul > li:nth-child(4) > a').click();
+
+        // 4. Verify 'New User Signup!' is visible
+        cy.get('.signup-form > h2').should('have.text','New User Signup!')
+
+        // 5. Enter name and email address
+        cy.get('input[name="name"]').type('gvantsa9');
+
+        cy.get('#form > div > div > div:nth-child(3) > div > form > input[type=email]:nth-child(3)').should('be.visible').type('gmamuchashvili19@gmail.com');
+
+        // 6. Click 'Signup' button
+        cy.get('#form > div > div > div:nth-child(3) > div > form > button').should('be.visible').click();
+
+        // 7. Verify that 'ENTER ACCOUNT INFORMATION' is visible
+        cy.get('#form > div > div > div > div > h2 > b').should('be.visible');

```

Comment:
ეცადე მსგავად არ დაწერო ხოლმე, რამე კლასით ან contains უკეთესი იქნებოდა და ეს სტეპი
Verify that 'ENTER ACCOUNT INFORMATION' is visible - არ სრულდება, იმავე მისამართზე ტექსტი რომ შეიცვალოს მაინც passed იქნება